Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -233,6 +233,53 @@ public extension UIView {
         }
     }
 
+    /// SwifterSwift: Return constraints for this view. This must enumerate superviews.
+    var allConstraints: [NSLayoutConstraint] {
+        // https://stackoverflow.com/questions/47053727
+
+        // build array of self and all superviews
+        var views = [self]
+        var current = self
+        while let superview = current.superview {
```

Comment:
Should this consider subviews here too?

Reply:
Not really sure ...

Reply:
I think constraints are always placed on the common ancestor (which is self or above). No need to check subviews.